cmd/kwil-admin/cmds/snapshot/create.go

```diff
@@ -0,0 +1,145 @@
+package snapshot
+
+import (
+	"compress/gzip"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	createLongExplain = `Creates a snapshot of the database. The command is used during network migration to get the state of the KwilDB.
+		It interacts directly with the Postgres server without intervening the kwild node.`
+)
+
+func createCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <db-name> <db-user> <db-host> <db-port> <snapshot-dir>",
+		Short: "Creates a snapshot of the database.",
+		Long:  createLongExplain,
+		Args:  cobra.ExactArgs(5),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
```

Comment:
We should use `cmd.Context()`. If the user cancels the command, we want the context to cancel too.